### PR TITLE
Add some JavaDoc to the maven plugin

### PR DESCRIPTION
**Pull request by dalexandrov**
Added some JavaDoc. Some parts are generated, but the most is handwritten.

### CODE REVIEW

File: maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/AbstractAddressMojo.java [hunk 1]
```diff
@@ -20,22 +20,43 @@
 import org.apache.maven.plugin.AbstractMojo;
 import org.apache.maven.plugins.annotations.Parameter;
 
+/**
+ * The type AbstractAddressMojo.
+ */
 public abstract class AbstractAddressMojo extends AbstractMojo {
+    /**
+     * The Tomee http port.
```
Comment (radcortez) [inline maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/AbstractAddressMojo.java]: Maybe use TomEE here.

File: maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/AbstractAddressMojo.java [hunk 2]
```diff
@@ -20,22 +20,43 @@
...
+     */
     @Parameter(property = "tomee-plugin.http")
     protected String tomeeHttpPort;
 
+    /**
+     * The Tomee https port.
```
Comment (radcortez) [inline maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/AbstractAddressMojo.java]: Maybe use TomEE here.

File: maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/AbstractAddressMojo.java [hunk 3]
```diff
@@ -20,22 +20,43 @@
...
+     */
     @Parameter(property = "tomee-plugin.https")
     protected String tomeeHttpsPort;
 
+    /**
+     * The Tomee host.
```
Comment (radcortez) [inline maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/AbstractAddressMojo.java]: Maybe use TomEE here.

File: maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/AbstractTomEEMojo.java [hunk 4]
```diff
@@ -99,37 +99,73 @@
...
     protected List<ArtifactRepository> remoteRepos;
 
+    /**
+     * The Skip current project.
+     */
     @Parameter(property = "tomee-plugin.skipCurrentProject", defaultValue = "false")
     protected boolean skipCurrentProject;
 
+    /**
+     * The Tomee version.
```
Comment (radcortez) [inline maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/AbstractTomEEMojo.java]: Maybe use TomEE here.

File: maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/AbstractTomEEMojo.java [hunk 5]
```diff
@@ -99,37 +99,73 @@
...
+     */
     @Parameter(property = "tomee-plugin.version", defaultValue = "-1")
     protected String tomeeVersion;
 
+    /**
+     * The Tomee group id.
```
Comment (radcortez) [inline maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/AbstractTomEEMojo.java]: Maybe use TomEE here.

File: maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/AbstractTomEEMojo.java [hunk 6]
```diff
@@ -99,37 +99,73 @@
...
+     */
     @Parameter(property = "tomee-plugin.groupId", defaultValue = "org.apache.tomee")
     protected String tomeeGroupId;
 
+    /**
+     * The Tomee artifact id.
```
Comment (radcortez) [inline maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/AbstractTomEEMojo.java]: Maybe use TomEE here.

File: maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/AbstractTomEEMojo.java [hunk 7]
```diff
@@ -148,45 +187,87 @@
     @Parameter(property = "tomee-plugin.classifier", defaultValue = "webprofile")
     protected String tomeeClassifier;
 
+    /**
+     * The Tomee shutdown port.
```
Comment (radcortez) [inline maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/AbstractTomEEMojo.java]: Maybe use TomEE here.

File: maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/AbstractTomEEMojo.java [hunk 8]
```diff
@@ -148,45 +187,87 @@
...
+     */
     @Parameter(property = "tomee-plugin.shutdown")
     protected String tomeeShutdownPort;
 
+    /**
+     * The Tomee shutdown attempts.
```
Comment (radcortez) [inline maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/AbstractTomEEMojo.java]: Maybe use TomEE here.

File: maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/AbstractTomEEMojo.java [hunk 9]
```diff
@@ -148,45 +187,87 @@
...
+     */
     @Parameter(property = "tomee-plugin.shutdown.attempts", defaultValue = "60")
     protected int tomeeShutdownAttempts;
 
+    /**
+     * The Tomee shutdown command.
```
Comment (radcortez) [inline maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/AbstractTomEEMojo.java]: Maybe use TomEE here.

File: maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/AbstractTomEEMojo.java [hunk 10]
```diff
@@ -148,45 +187,87 @@
...
+     */
     @Parameter(property = "tomee-plugin.shutdown-command", defaultValue = "SHUTDOWN")
     protected String tomeeShutdownCommand;
 
+    /**
+     * The Tomee ajp port.
```
Comment (radcortez) [inline maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/AbstractTomEEMojo.java]: Maybe use TomEE here.

File: maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/AbstractTomEEMojo.java [hunk 11]
```diff
@@ -283,57 +400,111 @@
...
     protected boolean checkStarted;
 
+    /**
+     * The Use console.
+     */
     @Parameter(property = "tomee-plugin.use-console", defaultValue = "true")
     protected boolean useConsole;
 
+    /**
+     * The Tomee already installed.
```
Comment (radcortez) [inline maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/AbstractTomEEMojo.java]: Maybe use TomEE here.

File: maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/AbstractTomEEMojo.java [hunk 12]
```diff
@@ -1439,14 +1662,29 @@ protected void serverCmd(final RemoteServer server, final List<String> strings)
...
+     * Gets connect attempts.
+     *
+     * @return the connect attempts
+     */
     protected int getConnectAttempts() {
         return (tomeeShutdownAttempts == 0 ? 60 : tomeeShutdownAttempts);
     }
 
+    /**
+     * Gets wait tom ee.
```
Comment (radcortez) [inline maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/AbstractTomEEMojo.java]: Maybe use TomEE here.

File: maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/AbstractTomEEMojo.java [hunk 13]
```diff
@@ -1439,14 +1662,29 @@ protected void serverCmd(final RemoteServer server, final List<String> strings)
...
+     *
+     * @return the wait tom ee
```
Comment (radcortez) [inline maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/AbstractTomEEMojo.java]: Maybe use TomEE here.

Comment (dalexandrov) [reply maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/AbstractTomEEMojo.java]: My bad! Definitely TomEE!

**Comment by radcortez** [comment]
Hey @dalexandrov!

Great to have you here :) Thank you for your PR.

If you could just replace the name TomEE instead of Tomee or other variations it should be good to go :)

**Comment by pzygielo** [comment]
What purpose is served by that kind of JavaDocs? I see only unneeded noise that duplicates what types/methods/fields/... **already** deliver with their names.

File: maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/Synchronization.java [hunk 14]
```diff
@@ -22,40 +22,83 @@
 import java.util.List;
 import java.util.Map;
 
+/**
+ * The type Synchronization to sunc the files and directories.
```
Comment (pzygielo) [inline maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/Synchronization.java]: What does it mean: `to sunc`?

File: maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/customizer/monkey/Monkey.java [hunk 15]
```diff
@@ -35,18 +35,26 @@
 /**
  * A customizer (see tomee-maven-plugin) allowing to take current project binaries (target/classes)
  * and merge them in tomee for patch cases.
- *
+ * <p>
```
Comment (pzygielo) [inline maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/customizer/monkey/Monkey.java]: No closing `p` tag.

File: maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/customizer/monkey/classloader/ClassLoaderFactory.java [hunk 16]
```diff
@@ -26,7 +26,17 @@
 import java.util.ArrayList;
 import java.util.Collection;
 
+/**
+ * The type Class loader factory is used to load classes drom the supplied *.jar and *.zip
```
Comment (pzygielo) [inline maven/tomee-maven-plugin/src/main/java/org/apache/openejb/maven/plugin/customizer/monkey/classloader/ClassLoaderFactory.java]: _drom_?

**Pull request closed without merging**